README.md

```diff
@@ -1,9 +1,127 @@
 # KubeSecondaryDNS
 DNS for KubeVirt VirtualMachines secondary interfaces
 
+## Prerequisites
+1. The KubeSecondaryDNS Deployment should be reachable from outside the cluster.
+2. The guest must have a guest agent installed, so the IP of the secondary interface will be reported.
+
```

Comment:
3. If necessary - establish connectivity to cluster public IP via relevant DNS entity that is used to reach the authoritative KubeSecondaryDNS server, such as DNSResolver, TLD NameServer, etc.

Reply:
Added thanks